Installing eoan fails differently from before

```
name: eoan
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/eoan...
crouton: version 1-20190807171712~master:6f45f06a
release: eoan
architecture: amd64
targets: lxde
host: version 12105.100.0 (Official Build) stable-channel celes 
kernel: Linux localhost 3.18.0-19346-g9ff80f5e4c97 #1 SMP PREEMPT Sun Jul 14 00:21:12 PDT 2019 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/eoan...
```

#### Please describe your issue:
Eoan fails to install properly (at least with lxde target). While compiling CRAS (i386), it gives an error and a note, then says `cc1: treating all warnings as errors` before failing. The errors are as follows:
```
libcras/cras_client.c: In function 'handle_message_from_server':
libcras/cras_client.c:1772:12: error: converting a packed 'struct cras_client_message' pointer (alignment 1) to a 'struct cras_client_get_hotword_models_ready' pointer (alignment 4) may result in an unaligned pointer value [-Werror=address-of-packed-member]
 1772 |    (struct cras_client_get_hotword_models_ready *)msg;
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from libcras/cras_client.c:50:
../src/common/cras_messages.h:89:37: note: defined here
   89 | struct __attribute__ ((__packed__)) cras_client_message {
      |                                     ^~~~~~~~~~~~~~~~~~~
../src/common/cras_messages.h:598:8: note: defined here
  598 | struct cras_client_get_hotword_models_ready {
      |        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
Failed to complete chroot setup.
Unmounting /mnt/stateful_partition/crouton/chroots/eoan...
```
This happens with both first install and update.

#### If known, describe the steps to reproduce the issue:
1.) Run `sudo crouton -r eoan -t lxde`
2.) Wait

--- a/targets/audio
+++ b/targets/audio
@@ -157,9 +157,11 @@ END
 	sed -i -e 's/cras_stream_id_t stream_id;/cras_stream_id_t stream_id = 0;/' \
 	    "$cras_test_client"
 
-	# FIXME(http://crrev.com/c/1759370): Remove when fixed upstream
+	# FIXME(http://crrev.com/c/1759370): Remove when R78 becomes minimum target
 	sed -i -e 's/\(strncpy.*sizeof(info.name)\));/\1 - 1); info.name[sizeof(info.name) - 1] = '"'"'\\0'"'"';/' \
 		common/cras_shm.c
+	sed -i -e 's/^struct \([^_]\)/struct __attribute__((__packed__)) \1/' \
+		common/cras_messages.h
 
         # Directory to install CRAS library/binaries
         CRASLIBDIR="/usr/local$archextrapath/lib"

